triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -73,7 +75,10 @@ class Runner(
   // Handles the result of initialState or update, i.e., (s, [Commands], Text)
   // by submitting the commands, printing the log message and returning
   // the new state
-  def handleStepResult(converter: Converter, v: SValue, submit: SubmitRequest => Unit): SValue =
+  private def handleStepResult(
+      converter: Converter,
```

Comment:
Does `converter` still need to be an argument when it's a member of `Runner`?

Reply:
It doesn’t, good catch! I’ll clean it up as part of the next refactoring PR.